packages/vue/src/components/SpCrud/SpCrud.vue

```diff
@@ -9,31 +9,47 @@
           size="md"
           style="font-weight: 700"
       >
-        Post items
+        {{ itemName }} items
       </SpTypography>
       <SpButton
-          aria-label="Create post"
           type="primary"
-          @click="visibleModal = 'create-post'"
+          @click="visibleModal = 'create-item'"
       >
-        Create post
+        Create {{ itemName }}
       </SpButton>
     </div>
 
     <SpSpacer size="sm" />
-
-    <SpCrudRead :storeName="storeName" />
+    <SpCrudRead
+      :storeName="storeName"
+      :itemName="itemName"
+      :commandName="`/Query${itemName}s`"
```

Comment:
Shouldn't it be `/Query${itemName}All`?

Reply:
ie = 

![image](https://user-images.githubusercontent.com/3693700/151424156-7679d377-ccb5-4b81-b9e3-17260ee1cfb2.png)

Reply:
current = 

https://github.com/tendermint/vue/blob/8592c94e43592b17c7367606ace161d80a9fff74/packages/vue/src/components/SpTypeList/SpTypeList.vue#L122

Reply:
generated = 
![image](https://user-images.githubusercontent.com/3693700/151425096-0ab9fcf6-42e6-4b8a-a598-f419dff05a7d.png)

Reply:
`QueryBlogAll` doesn't work... `QueryBlogs` work.

Reply:
@Matthews3301 how does it work, though, if there is no `Query${item}s` action?